src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/ServiceAgencySwitchResourceIntTest.kt

```diff
@@ -33,13 +33,14 @@ class ServiceAgencySwitchResourceIntTest : ResourceTest() {
 
   private lateinit var leeds: AgencyLocation
   private lateinit var moorland: AgencyLocation
+  private lateinit var someService: ExternalService
 
   @BeforeEach
   fun `set up`() {
     leeds = agencyLocationRepository.findByIdOrNull("LEI") ?: throw EntityNotFoundException("Agency LEI is not saved")
     moorland = agencyLocationRepository.findByIdOrNull("MDI") ?: throw EntityNotFoundException("Agency MDI is not saved")
 
-    val someService = externalServiceRepository.save(ExternalService.builder().serviceName("SOME_SERVICE").description("Some service").build())
+    someService = externalServiceRepository.save(ExternalService.builder().serviceName("SOME_SERVICE").description("Some service").build())
```

Comment:
this is a bit odd - isn't `ExternalService` new?  Why isn't it then written in kotlin instead?

Reply:
Because I'm lazy and didn't want to work out how to do an AuditableEntity superclass. But you're right, I'll do it.